crates/matrix-sdk-crypto/src/types/events/mod.rs

```diff
@@ -42,7 +42,7 @@ pub trait EventType {
     ///
     /// **Note**: This should never be implemented manually, this takes the
     /// event type from the constant.
```

Comment:
This is no longer true? Or at least, the word "never" should be softened to "not usually" or something.

You could fix the comma splice while you're there.

Reply:
Did so in 49d119a397b53e45b81197904c09a83687f503b4.